toronto/src/components/molecules/Upload/index.js

```diff
@@ -0,0 +1,117 @@
+import styled from 'styled-components';
+import { useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+
+const UploadContainer = styled.div`
+  display: inline-block;
+`;
+
+const Input = styled.input`
+  display: none;
+`;
+
+const Upload = ({
+  children,
+  clickArea = false,
+  droppable,
+  name,
+  accept,
+  value,
+  onChange,
+  ...props
+}) => {
+  const [file, setFile] = useState(value);
+  const [dragging, setDragging] = useState(false);
+  const inputRef = useRef(null);
+  const enterCounter = useRef(0);
+
+  const handleFileChange = (e) => {
+    const files = e.target.files;
+    const changedFile = files[0];
+    setFile(changedFile);
+    onChange && onChange(changedFile);
+  };
+
+  const handleChooseFile = () => {
+    inputRef.current.click();
```

Comment:
ref에 참조되기 전에 발생될 가능성이 높아서, 여기는 가드를 하셔야할거같아요
클릭되면 click 핸들러를 참조가 안될거라서요. 
```suggestion
    inputRef.current?.click();
```
```suggestion
    if(!inputRef.current) return;
```